src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -1066,13 +1066,17 @@ protected void ExecuteIndexing()
                                     if (_logger.IsOperationsEnabled)
                                         _logger.Operations($"Failed to open write transaction, indexing will be retried", te);
                                 }
+                                catch (InsufficientMemoryException ime)
+                                {
+                                    HandleOutOfMemoryException(ime, scope, $"{_environment}:{nameof(InsufficientMemoryException)}");
```

Comment:
We are in low memory already, and you are allocating additional string.
Let's create this once